CONTRIBUTING.md

```diff
@@ -30,8 +30,8 @@ Here's a high-level overview of the repository structure:
 
 For an in-depth guide on how to set up your developer environment and add an agentic action, see the following language-specific guides:
 
-- [Python Development Guide](./CONTRIBUTING-PYTHON.md)
-- [TypeScript Development Guide](./CONTRIBUTING-TYPESCRIPT.md)
+- [Python Development Guide](./.github/CONTRIBUTING-PYTHON.md)
```

Comment:
You need to move the CONTRIBUTING-PYTHON and CONTRIBUTING-TYPESCRIPT files as well